src/ifc-printer/printer.cxx

```diff
@@ -211,7 +211,7 @@ namespace ifc::util {
             std::string indent_strs[static_cast<int>(IndentAction::Recurse_last) + 1]{"|-", "\\-", "| ", "  "};
         };
 
-        TreePrinter& TreePrinter::operator<<(std::string_view str)
+        [[maybe_unused]] TreePrinter& TreePrinter::operator<<(std::string_view str)
```

Comment:
Is GCC warning on the absence of this attribute?  Why would it do that?

Reply:
I built with LLVM Clang 17 in my Linux VM and it warned about the function being unused. I was deliberating just deleting the function, but went with the attribute instead.

Reply:
Yeah, if it is unused, when just remove it.